app/src/test/kotlin/com/getstrm/pace/processing_platforms/postgres/PostgresViewGeneratorTest.kt

```diff
@@ -176,10 +198,7 @@ where (
     when ('fraud_and_risk' IN ( SELECT rolname FROM user_groups )) then true
     else (ts + 10 * interval '1 day') > current_timestamp
   end
-  and case
-    when ('fraud_and_risk' IN ( SELECT rolname FROM user_groups )) then (validThrough + 365 * interval '1 day') > current_timestamp
-    else (validThrough + 0 * interval '1 day') > current_timestamp
-  end
+  and (validThrough + 365 * interval '1 day') > current_timestamp
```

Comment:
What happened here?

Reply:
Needed an example without the `case when` so I removed one of the conditions because it is already covered in the same test